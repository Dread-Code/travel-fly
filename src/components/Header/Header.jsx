import React from 'react'
import { GiCrossedAirFlows } from 'react-icons/gi'
import './Header.scss'

const Header = () => (
  <header className="header">
    <div className="header__logo">
      <GiCrossedAirFlows />
      <div>
        <h1>Travel Fly</h1>
      </div>
    </div>
    <div />
    <div>
      <h2 className="header__description">Las Mejores Aerolineas a los Mejores Precios</h2>
    </div>
  </header>
)

export default Header
