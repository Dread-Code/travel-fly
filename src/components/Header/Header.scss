.header {
  display: flex;
  width: 100%;
  justify-content: space-around;
  background: #ddd5f5;
  position: relative;
  align-items: center;
  padding: 2vw 0;
  z-index: 1;
  -webkit-box-shadow: 0px 4px 8px -2px #540fed;
  box-shadow: 0px 4px 8px -2px #540fed;
  &__logo {
    color: #540fed;
    font-size: 7vw;
    display: flex !important;
    align-items: center;
  }

  &__description {
    font-size: 2vw;
  }
}

@media (min-width: 580px) {
  .header {
    &__logo {
      font-size: 5vw;
    }
  }
}

@media (min-width: 992px) {
  .header {
    height: 2vw;
    &__logo {
      font-size: 3vw;
    }
    &__description {
      font-size: 1vw;
    }
  }
}
