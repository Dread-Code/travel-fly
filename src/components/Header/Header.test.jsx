import { shallow } from 'enzyme'
import React from 'react'
import Header from './Header'

describe('Header', () => {
  test('should ', () => {
    const wrapper = shallow(<Header />)
    expect(wrapper).toMatchSnapshot()
  })
})
