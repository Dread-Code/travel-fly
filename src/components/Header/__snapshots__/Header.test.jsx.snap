// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should  1`] = `
<header
  className="header"
>
  <div
    className="header__logo"
  >
    <GiCrossedAirFlows />
    <div>
      <h1>
        Travel Fly
      </h1>
    </div>
  </div>
  <div />
  <div>
    <h2
      className="header__description"
    >
      Las Mejores Aerolineas a los Mejores Precios
    </h2>
  </div>
</header>
`;
